airflow/contrib/operators/dataproc_operator.py

```diff
@@ -55,7 +55,7 @@ def __init__(self,
                  num_preemptible_workers=0,
                  labels=None,
                  region='global',
-                 google_cloud_conn_id='google_cloud_default',
+                 gcp_conn_id='google_cloud_default',
```

Comment:
a bit concerned about this change since it's not backwards compatible (for those who explicitly set `google_cloud_conn_id`.